lib/trento/hosts/host.ex

```diff
@@ -262,7 +272,8 @@ defmodule Trento.Hosts.Host do
         fully_qualified_domain_name: fully_qualified_domain_name,
         installation_source: installation_source
       }
-    ]
+    end)
+    |> Multi.execute(&maybe_emit_software_updates_discovery_requested_event/1)
```

Comment:
In this scenario, we have the chance of having a `fqdn` before, but the new resister that restores sends a nil `fqdn`, in which case, i guess we should emit `SoftwareUpdatesDiscoveryCleared`.

PD: with the proposal I did, this should be covered, as we only have one function

Reply:
during a restoration, the transition of the fqdn to null does not really require a clean up, because things were cleared on deregistration. makes sense?

Reply:
yes, you are right, it makes sense.